addons/web_editor/__manifest__.py

```diff
@@ -86,13 +85,24 @@
             'web_editor/static/src/js/wysiwyg/wysiwyg.js',
             'web_editor/static/src/js/wysiwyg/wysiwyg_iframe.js',
         ],
+        'web_editor.assets_media_dialog': [
```

Comment:
The assets_media_dialog xml templates need to be included in the `web.assets_frontend` bundle, because only this bundle is fetched to process the templates for the frontend

Reply:
I don't understand how this answers my question? :thinking: :wink: I must be missing something.

What I see is this:
![image](https://user-images.githubusercontent.com/10338094/173329851-59526c23-8d11-4cc3-a373-26cf18cb1771.png)

1. Definition
2. Addition in backend
3. Addition in frontend
4. Extension
5. Extension

Why can't it be:
1. Addition in common
2. Extension (in common)
3. Extension (in common)

?

Reply:
Sorry, I should have been more explicit, the assets_media_dialog bundle includes xml files with owl templates,
The public root is created with only the owl templates included in 'web.assets_frontend': https://github.com/odoo/odoo/blob/a0e9e5670d84a4c559ec200aaac7b5f9dc054918/addons/web/static/src/legacy/js/public/public_root.js#L408
So maybe that should be changed to fetch and process the templates from the 'web.assets_common' bundle ?